view_student.html

```diff
@@ -5,8 +5,91 @@
   <head>
   <title>View Student</title>
   <link rel="stylesheet" type="text/css" href="cs/upls_styles.css">
-  <script src="js/main_menu.js"></script>
+  <script src="js/main_menu.js"></script> 
+  <script src="https://apis.google.com/js/api.js"></script>
+  <script src="https://apis.google.com/js/platform.js?onload=init" async defer></script>
+  <script src="functions.js"></script>
+  <script>
+    let sa = SheetsApi("1n2w0s1lqSZ4kHX3zeNYT-UNRPEr1aextWaG_bsJisn8","AIzaSyDeampVGzzd8NvBiUtEsNVmNkAQU1TZ17I","21358841826-edt9rotek8r1rbivt91nabpn2sc2g6ts.apps.googleusercontent.com");
+    sa.handleClientLoad();	
+  </script>
   </head>
+  <body class='body'>
+	<script type="text/javascript">
```

Comment:
Move this to a separate js file too.

Reply:
Remember to include a signin check as well. Like what Zeru has in edit_header.js